packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -382,3 +395,39 @@ enum SkuType {
   @JsonValue('subs')
   subs,
 }
+
+/// Enum representing the proration mode. When upgrading or downgrading a subscription,
```

Comment:
nits: dart doc should start with a single sentence summary and an empty line between summary and description. See: https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary
ditto the docs for all the enum values.

Reply:
Done